charts/incubator/warrior/values.yaml

```
@@ -0,0 +1,35 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: atdr.meo.ws/archiveteam/warrior-dockerfile
+  # -- image tag
+  tag: latest
```

```suggestion
  # -- image tag
  # @default -- chart.appVersion
  tag:
```
This makes it so that we only have to update the appVersion (by default) but it still allows users to override it if they want.